app/ide-desktop/lib/dashboard/src/utilities/__tests__/dateTime.test.ts

```diff
@@ -9,18 +9,23 @@ import * as dateTime from '#/utilities/dateTime'
 
 /* eslint-disable @typescript-eslint/no-magic-numbers */
 
-/** The number of milliseconds in an hour. */
-const HOUR_MS = 3_600_000
-/** The number of minutes in an hour. */
-const HOUR_MIN = 60
-const TIMEZONE_OFFSET_MINS = new Date().getTimezoneOffset()
-/** The offset of local time */
-const TIMEZONE_OFFSET_HOURS = Math.floor(TIMEZONE_OFFSET_MINS / HOUR_MIN)
+/** The number of milliseconds in a minute. */
+const MIN_MS = 60_000
+/** Remove all UTC offset from a {@link Date}. Daylight savings-aware. */
```

Comment:
I don’t think convertation from UTC to local involves *removing* offset. We actually add one, and the code matches. Same for `convertLocalToUTC` – we don’t add the offset, we remove one.

Reply:
uh oh. yeah good catch, the docs are from a prior time where i was confused (before i fixed the names but not the docs)